sdk/v1/client.go

```diff
@@ -92,6 +92,31 @@ func (cli *Client) PostJob(ctx context.Context, headers client.Headers) (models.
 	return job, nil
 }
 
+// PostTasksCount Updates tasks count for processing
+func (cli *Client) PostTasksCount(ctx context.Context, headers client.Headers, jobID string, payload []byte) (models.Task, error) {
```

Comment:
I think this should be a TaskToCreate
```suggestion
func (cli *Client) PostTasksCount(ctx context.Context, headers client.Headers, jobID string, taskToCreate models.TaskToCreate) (models.Task, error) {
```

Reply:
nope, the client will be invoked with payload as a byte slice. please let me know if need to explain further.